src/components/UI/FullScreenButton/FullScreenButton.test.js

```diff
@@ -18,26 +18,36 @@ jest.mock('./FullScreenButton.messages', () => {
 });
 
 describe('FullScreenButton tests', () => {
+  beforeEach(() => {
+    document.exitFullscreen = jest.fn();
+    document.documentElement.requestFullscreen = jest.fn();
+  });
+
   test('default renderer', () => {
     shallowMatchSnapshot(<FullScreenButton onClick={() => {}} />);
   });
   test('on buttton click', () => {
     const wrapper = mount(
-      shallow(<FullScreenButton disabled={false} />).get(0)
+      <FullScreenButton
+        disabled={false}
+        intl={{ formatMessage: msg => msg.defaultMessage }}
+      />
     );
-    wrapper.simulate('click');
-    expect(wrapper.state().fullscreen).toEqual(true);
-    wrapper.simulate('click');
-    expect(wrapper.state().fullscreen).toEqual(false);
+    const button = wrapper.find('button');
+    button.simulate('click');
+    expect(document.documentElement.requestFullscreen).toHaveBeenCalled();
+    button.simulate('click');
+    expect(document.exitFullscreen).toHaveBeenCalled();
   });
   test('on buttton click', () => {
-    global.window.document.requestFullscreen = jest.fn();
+    global.window.document.fullscreenElement = jest.fn();
     const wrapper = mount(
       shallow(<FullScreenButton disabled={false} />).get(0)
     );
-    wrapper.simulate('click');
-    expect(wrapper.state().fullscreen).toEqual(true);
-    wrapper.simulate('click');
-    expect(wrapper.state().fullscreen).toEqual(false);
+    const button = wrapper.find('button');
+    button.simulate('click');
+    expect(document.documentElement.requestFullscreen).toHaveBeenCalled();
+    button.simulate('click');
+    expect(document.exitFullscreen).toHaveBeenCalled();
   });
```

Comment:
This test is the same than the above right? If that is the case, please remove it